### PR TITLE
Feature/player move api

### LINKED ISSUE #48: Feature/player-move-api

### PR DESCRIPTION
**Pull request by zaktolba**

### CODE REVIEW

File: src/components/Party/Party.vue [hunk 1]
```diff
@@ -63,18 +65,29 @@ export default {
...
-      const players = this.players.map(pl => pl)
-      const playerIndex = players.indexOf(player)
-      players[playerIndex] = { ...player, x: position.x, y: position.y }
-
-      this.$set(this.$data, 'players', players)
+    async movePlayer(player, position) {
+      try {
+        const URL = 'https://wars.quiches.ovh/api/party/action'
+        const result = await this.$http.put(URL, {
+          "partyId": this.partyId,
```
Comment (MaxenceMottard) [inline src/components/Party/Party.vue]: remove " please

File: src/components/Party/Party.vue [hunk 2]
```diff
@@ -35,6 +36,7 @@ export default {
   },
   data() {
     return {
+      partyId : String(this.$route.params.partyId),
```
Comment (MaxenceMottard) [inline src/components/Party/Party.vue]: Is "String()" really necessary ?

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

File: src/components/Party/Party.vue (hunk 2 above)
Comment (zaktolba) [reply src/components/Party/Party.vue]: Inspired by CreateParty and JoinPartyStep1, apparently not necessary for this case

**Review by MaxenceMottard — APPROVED** [review]

File: src/components/Party/Party.vue [hunk 3]
```diff
@@ -63,18 +65,29 @@ export default {
...
     },
-    movePlayer(player, position) {
-      const players = this.players.map(pl => pl)
-      const playerIndex = players.indexOf(player)
-      players[playerIndex] = { ...player, x: position.x, y: position.y }
-
-      this.$set(this.$data, 'players', players)
+    async movePlayer(player, position) {
+      try {
+        const URL = 'https://wars.quiches.ovh/api/party/action'
```
Comment (Laroche-Nicolas) [inline src/components/Party/Party.vue]: you can use variable from .env file to set URL

Comment (zaktolba) [reply src/components/Party/Party.vue]: API_BASE_URL is throwing undefined

**Review by Laroche-Nicolas — APPROVED** [review]

**Pull request merged**